.github/workflows/build-test.yaml

```diff
@@ -31,3 +31,45 @@ jobs:
 
       - name: Build Tracee
         run: make build-docker
+  
+  test:
+    runs-on: ubuntu-20.04
+
+    env:
+      CGO_ENABLED: "1"
+
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+        id: go
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Lint
+        run: |
+          if test -z "$(gofmt -l .)"; then
+            echo "Congrats! There is nothing to fix."
+          else
+            echo "The following lines should be fixed."
+            gofmt -s -d .
+            exit 1
+          fi
+
+      - name: Install Libraries
+        run: |
+          sudo apt-get update
+          sudo apt install -y bison build-essential cmake flex git libedit-dev \
+              libllvm8 llvm-8-dev libclang-8-dev libllvm9 llvm-9-dev libclang-9-dev \
+              python zlib1g-dev libelf-dev libtinfo5 libfl-dev
+          git clone https://github.com/iovisor/bcc.git
```

Comment:
please rebase on latest changes.
bcc is not a dependency anymore (I guess these 5 lines can now be removed)